Services/ISSLCertificateService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Orchard.SSLCertificate.Settings;
+using Orchard.SSLCertificate.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Orchard.SSLCertificate.Services
+{
+    public interface ISSLCertificateService
+    {
+        SSLCertificateSettings GetSSLCertificateSettings();
+        void UpdateSSLCertificateSettings(SSLCertificateSettings SSLCertificateSettings);
+        bool IsValidSSLCertificateSettings(SSLCertificateSettings SSLCertificateSettings);
+        bool IsValidSSLCertificateSettings(SSLCertificateSettings settings, ModelStateDictionary modelState);
```

Comment:
choose a name for the parameter, SSLCertificateSettings, or just settings, but name it always on the same way